salt/config/__init__.py

```diff
@@ -2225,13 +2220,16 @@ def load_config(path, env_var, default_path=None, exit_on_config_errors=True):
 
     # Default to the environment variable path, if it exists
     env_path = os.environ.get(env_var, path)
+    if env_path is not None:
+        salt.utils.path.expand_path(env_path)
```

Comment:
What that supposed to do?

Reply:
I just add a comment

```python
    # Default to the environment variable path, if it exists
    env_path = os.environ.get(env_var, path)
    if env_path:
        # Allow path like '~/foo' and expand 'foo' or './foo'
        # relatively to root_dir
        env_path = salt.utils.path.expand_path(env_path)
```

I added the same comment in `salt.config.include_config`.

Reply:
I know what it does. But you never redefined `env_path` variable originally, so it was just a no-op call:
```python
 if env_path is not None:
        salt.utils.path.expand_path(env_path)
```